app/assets/javascripts/decidim/admin/blank_vote.js

```diff
@@ -0,0 +1,35 @@
+function blankVote(element) {
+  let translations = JSON.parse(element.getAttribute("value"));
+
+  if (element.checked === true) {
+    getInputTitles(true, translations);
+  } else {
+    getInputTitles(false, translations);
+  }
+}
+
+function getInputTitles (blankVote, translations) {
+  const languageTitles = document.getElementById('response-title-tabs').getElementsByTagName('li');
+
+  for (let i = 0; i < languageTitles.length; i++) {
```

Comment:
Sorry but I missed check this part of code applying the same criteria.
In this case is better to get the different languages from de data getted from the blank vote checkbox field. With this you can set all locale title fields you need:

`  
for(let i=0; i< Object.keys(translations.data).length; i++) {
    putTitle(Object.values(translations.data)[i], Object.keys(translations.data)[i], blankVote);
  }
`
(I don't know why the code render not properly rendering my code in the comment)